README.md

```diff
@@ -1,2 +1,147 @@
 # flask-monitor
 A Prometheus middleware to add basic but very useful metrics for your Python Flask app.
+
+# Metrics
+
+As valid Big Brother library it exposes the following metrics:
+
+```
+request_seconds_bucket{type, status, isError, method, addr, le}
+request_seconds_count{type, status, isError, method, addr}
+request_seconds_sum{type, status, isError, method, addr}
+response_size_bytes{type, status, isError, method, addr}
+dependency_up{name}
+application_info{version}
+```
+
+Where, for a specific request, `type` tells which request protocol was used (e.g. `grpc` or `http`), `status` registers the response HTTP status, `method` registers the request method, `addr` registers the requested endpoint address, `version` tells which version of your app handled the request and `isError` lets us know if the status code reported is an error or not.
+
+In detail:
+
+1. The `request_seconds_bucket` metric defines the histogram of how many requests are falling into the well defined buckets represented by the label `le`;
+
+2. The `request_seconds_count` is a counter that counts the overall number of requests with those exact label occurrences;
+
+3. The `request_seconds_sum` is a counter that counts the overall sum of how long the requests with those exact label occurrences are taking;
+
+4. The `response_size_bytes` is a counter that computes how much data is being sent back to the user for a given request type. It captures the response size from the `content-length` response header. If there is no such header, the value exposed as metric will be zero;
+
+5. The `dependency_up` is a metric to register weather a specific dependency is up (1) or down (0). The label `name` registers the dependency name;
+
+6. Finally, `application_info` holds static info of an application, such as it's semantic version number;
+
+## Labels
+
+For a specific request:
+
+1. `type` tells which request protocol was used (e.g. `grpc`, `http`, `<your custom protocol>`);
+2. `status` registers the response status code; 
+3. `isError` let you know if the request's response status is considered an error;
+4. `method` registers the request method (e.g. `GET` for http get requests);
+5. `addr` registers the requested endpoint address;
+6. and `version` tells which version of your service has handled the request;
+
```

Comment:
@gutorc92, this seems redundant with the text from the previous section. Please, choose only one.